.github/workflows/spellcheck.yml

```diff
@@ -0,0 +1,23 @@
+name: Spell Check
+
+on: [push, pull_request]
+
+jobs:
+  spellcheck:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: Install Codespell
+        run: pip install codespell
```

Comment:
@youyyytrok  re-running the same CI job started failing: 
– https://github.com/ipfs/kubo/actions/runs/12788199550/job/36008198868#step:4:10

 I think this is due to codespell version not being pinned – any way we can control this so it does not break CI every time codespell updates rules?